products/epinio/list/services.vue

```diff
@@ -11,7 +11,9 @@ export default {
   },
   async fetch() {
     this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.APP });
-    await this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.SERVICE });
+
+    // Force refetch so list is updated when service is created
+    await this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.SERVICE, opt: { force: true } });
```

Comment:
We shouldn't force on visiting the list, rather once we've successfully created a service

Reply:
I've updated the PR so that it uses the nuxt event bus to know when to force-update the list. The list component creates an event listener to listen for service creation. Then when when you create a service, it emits the event.